vpr/src/device/rr_graph_builder.h

```diff
@@ -37,6 +37,15 @@ class RRGraphBuilder {
     t_rr_graph_storage& node_storage();
     /* Return a writable object for update the fast look-up of rr_node */
     RRSpatialLookup& node_lookup();
+    /* Add an existing rr_node in the node storage to the node look-up
```

Comment:
I think this comment should also say that the node will be added to the lookup for every side it is on (for OPINs and IPINs) and for every (x,y) location at which it exists (for wires that span more than one (x,y).